oada/services/auth/token.js

```diff
@@ -0,0 +1,139 @@
+const argv = require('minimist')(process.argv.slice(2));
+const cloneDeep = require('clone-deep');
+const chalk = require('chalk');
+const uuid = require('uuid');
+
+const { authorizations, users } = require('@oada/lib-arangodb');
+
+if (!process.env.DEBUG) {
+  process.env.DEBUG = 'token:info,token:error';
```

Comment:
A CLI tool should just be directly printing anything for user interaction rather than forcing `debug` to display it,